packages/client/src/forms/register/mappings/query/registration-mappings.ts

```diff
@@ -138,7 +139,7 @@ export const localPhoneTransformer =
   }
 
 const getUserFullName = (history: History): string => {
-  return history?.user ? getUserName(history.user) : ''
+  return history?.user?.name ? createNamesMap(history.user.name)[LANG_EN] : ''
```

Comment:
Is this a bug fix to some issue in the flow?

Reply:
It's not actually. `getUserName` was one of the transformer functions related to pdfmake that I removed in this PR. So had to refactor